tools/osx/macosx_bundle.sh

```diff
@@ -110,7 +110,9 @@ EXPATLIB="$(cmake .. -LA -N | grep "pkgcfg_lib_EXPAT_expat" | cut -d "=" -f2)"
 CODESIGNID="$(cmake .. -LA -N | grep "CODESIGNID" | cut -d "=" -f2)"
 NOTARY="$(cmake .. -LA -N | grep "NOTARY" | cut -d "=" -f2)"
 FANCY_DMG="$(cmake .. -LA -N | grep "FANCY_DMG" | cut -d "=" -f2)"
-echo ${FANCY_DMG}
+if [[ -n ${FANCY_DMG} ]] ; then
```

Comment:
No need for curly braces.

Reply:
Curly braces and quotes removed from strings in [[ statements.